src/Pages/HallOfFamePage/index.jsx

```diff
@@ -1,6 +1,8 @@
 import React, { useState } from "react";
-import Navbar from "../Components/Navbar";
-import * as s from "../Styles/HallOfFamePageStyle";
+import Navbar from "../../Components/Navbar/index";
```

Comment:
폴더 안의 index 파일은 기본 파일이라 생략해도 괜찮을거 같습니다!
```suggestion
import Navbar from "../../Components/Navbar";
```

Reply:
생략 완료 했습니다!